helm/templates/deployment.yaml

```diff
@@ -91,6 +93,8 @@ spec:
             - "$(KRO_CLIENT_BURST)"
             {{- if .Values.config.enableLeaderElection }}
             - --leader-elect
+            - --leader-election-namespace
```

Comment:
nit: im thinking its probably safer to no set this arg at all if empty because otherwise the value would be incorrectly parsed when there is a follow up arg. WDYT?